### PR TITLE
(feat) O3-3244 & O3-3250: Add hook for fetching EMR Configuration

### PR DESCRIPTION
**Pull request by vasharma05**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-docs.openmrs.org/docs/frontend-modules/contributing.en-US#contributing-guidelines) label. See existing PR titles for inspiration.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://om.rs/o3ui).
- [x] My work includes tests or is validated by existing tests.

## Summary
This PR adds a new hook `useEmrConfiguration` to fetch the EMR Configuration.

## Screenshots
None

## Related Issue
https://issues.openmrs.org/browse/O3-3244
https://issues.openmrs.org/browse/O3-3250

### CODE REVIEW

**Review by usamaidrsk — APPROVED** [review]
Thanks @vasharma05 , Looks good!

File: packages/esm-ward-app/src/hooks/useEmrConfiguration.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  clinicianEncounterRole: OpenmrsResource;
+  consultFreeTextCommentsConcept: OpenmrsResource;
+  visitNoteEncounterType: OpenmrsResource;
+  // There are many more keys to this object, but we only need these for now
+  // Add more keys as needed
+}
+
+export default function useEmrConfiguration() {
+  const swrData = useSWRImmutable<FetchResponse<EmrApiConfigurationResponse>>(
+    `${restBaseUrl}/emrapi/onfiguration`,
```
Comment (denniskigen) [inline packages/esm-ward-app/src/hooks/useEmrConfiguration.ts]:
```suggestion
    `${restBaseUrl}/emrapi/configuration`,
```

typo?

File: packages/esm-service-queues-app/translations/en.json [hunk 2]
```diff
@@ -275,6 +275,7 @@
   "ticketNumber": "Ticket Number",
   "time": "Time",
   "timeCannotBeInFuture": "Time cannot be in the future",
+  "timeCannotBePriorToPreviousQueueEntry": "Time cannot be before start of previous queue entry: {{time}}",
   "timeOfTransition": "Time of transition",
   "tirageNotYetCompleted": "Triage has not yet been completed",
```
Comment (denniskigen) [inline packages/esm-service-queues-app/translations/en.json:280]:
```suggestion
  "triageNotYetCompleted": "Triage has not yet been completed",
```

Comment (vasharma05) [reply packages/esm-service-queues-app/translations/en.json:280]: Hi @denniskigen!
This change in translation is not related to this PR, the change came up from the `extract-translations` command. Let me get another PR to improve the key name.
Thanks!

File: packages/esm-ward-app/src/hooks/useEmrConfiguration.ts (hunk 1 above)
Comment (vasharma05) [reply packages/esm-ward-app/src/hooks/useEmrConfiguration.ts]: Actually, I was testing failure case and forgot to fix it, I have fixed it now.
Thanks for the catch!

**Review by mogoodrich — COMMENTED** [review]
Belated LGTM, thanks @vasharma05 !

**Pull request merged**